recipes/impute5/meta.yaml

```diff
@@ -0,0 +1,31 @@
+package:
+  name: impute5
+  version: '1.1.5'
+
+source:
+  url: https://github.com/LouisLeNezet/Impute5/raw/main/impute5_v1.1.5.zip
```

Comment:
If you're not the author of this tool, is the author aware of this mirror? It would be much better for them to host and maintain themselves.